components/gallery/GalleryItem.vue

```diff
@@ -96,11 +99,39 @@
       data-cy="carousel-related" />
 
     <CarouselTypeVisited class="mt-6" />
+
+    <b-modal
+      v-model="isFullscreen"
+      :destroy-on-hide="false"
+      :can-cancel="false"
+      full-screen
+      custom-class="gallery-item-modal"
+      custom-content-class="gallery-item-modal-content">
+      <NeoButton class="back-button" @click.native="isFullscreen = false">
+        <NeoIcon icon="chevron-left" />
+        {{ $i18n.t('go back') }}
+      </NeoButton>
+
+      <div class="gallery-item-modal-container">
+        <MediaItem
+          :key="nftImage"
+          :class="{
+            'is-flex is-align-items-center is-justify-content-center h-audio':
+              resolveMedia(nftMimeType) == MediaType.AUDIO,
+          }"
+          class="gallery-item-media"
+          :src="nftImage"
+          :animation-src="nftAnimation"
+          :mime-type="nftMimeType"
+          :title="nft?.name || nft?.id"
+          original />
+      </div>
+    </b-modal>
```

Comment:
could you make it as a component?
why do we have a part related with audio media?

Reply:
@floyd-li ye, that makes sense, audio only has cover img that does not really need fullscreen button tbh